Content.Server/TapeRecorder/CassetteTapeComponent.cs

```diff
@@ -0,0 +1,37 @@
+using System.Threading;
+
+namespace Content.Server.TapeRecorder
+{
+
+    /// <summary>
+    /// Component given to cassette tapes, these hold the information recorded onto them by tape recorders.
+    /// </summary>
+    [RegisterComponent]
+    public sealed class CassetteTapeComponent : Component
+    {
+        /// <summary>
+        /// A list of all recorded messages with timestamps
+        /// </summary>
+        [ViewVariables(VVAccess.ReadOnly)]
+        public List<(float MessageTimeStamp, string Message)> RecordedMessages = new ();
+
+        public CancellationTokenSource? CancelToken = null;
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("unspooled")]
+        public bool UnSpooled = false;
```

Comment:
inconsistent casing between the datafield name and C# name. unspooled is one word.